runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -82,7 +82,13 @@ void verboseOptions(compiler::CompilerOptions &options)
   VERBOSE(Compiler) << "fp16_enable              : " << options.fp16_enable << std::endl
                     << std::noboolalpha;
 }
+} // namespace
 
+namespace onert
+{
+
+namespace compiler
+{
```

Comment:
Now, `setBackendMap` is in namespace onert::compiler to be used in `nnfw_api_internal.cc`.